libcaf_net/caf/internal/lp_flow_bridge.cpp

```diff
@@ -2,31 +2,31 @@
 // the main distribution directory for license terms and copyright or visit
 // https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
 
-#include "caf/detail/lp_flow_bridge.hpp"
+#include "caf/internal/lp_flow_bridge.hpp"
 
 #include "caf/net/lp/lower_layer.hpp"
 #include "caf/net/lp/upper_layer.hpp"
 #include "caf/net/socket_manager.hpp"
 
 #include "caf/async/blocking_producer.hpp"
 #include "caf/cow_tuple.hpp"
-#include "caf/detail/flow_bridge_base.hpp"
+#include "caf/internal/flow_bridge_base.hpp"
 
 #include <cstddef>
 
-namespace caf::detail {
+namespace caf::net::lp {
```

Comment:
Any particular reason you didn't use `caf::internal` here?

Reply:
Anything in this block goes into the anonymous namespace anyways, so I've picked the most convenient namespace for the implementation.